modules/nodes-cluster-features-about.adoc

```diff
@@ -3,31 +3,44 @@
 // * nodes/nodes-cluster-enabling-features.adoc
 
 [id="nodes-cluster-features-about_{context}"]
-= Understanding feature gates and Technology Preview features
+= Understanding feature gates
 
-You can use the Feature Gate Custom Resource to enable Technology Preview
-features throughout your cluster. This allows you, for example, to enable
-Technology Preview features on test clusters where you can fully test them while
-ensuring they are disabled on production clusters.
+You can use the Feature Gate custom resource (CR) to enable specific feature sets throughout your cluster. A feature set is a specific collection of {product-title} features. 
 
-[IMPORTANT]
-====
-After turning Technology Preview features on using feature gates, they cannot be
-turned off and cluster upgrades are prevented.
+For example, the `TechPreviewNoUpgrade` feature set contains all Technology Preview features. This feature set allows you to enable a subset of the current Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters.
 
-ifndef::openshift-origin[]
-For more information about the support scope of Red Hat Technology Preview features,
-see link:https://access.redhat.com/support/offerings/techpreview/[].
-endif::[]
-====
+You can activate any of the following feature sets by using the Feature Gate (CR):
 
+[options="header"]
+|===
+| FeatureSet| Description
 
-////
-If you disable a feature that appears in the web console, you might see that feature, but
-no objects are listed. For example, if you disable builds, you can see the *Builds* tab in the web console, but there are no builds present.
+|`IPv6DualStackNoUpgrade`
+|Enables the dual-stack networking mode in your cluster. Dual stack networking supports the use of IPv4 and IPv6 simultaneously. Turning on this feature set on is not supported, cannot be undone, and prevents upgrades. 
+
+|`LatencySensitive`
+|Enables the Topology Manager in your cluster. Upgrades are enabled with this feature.
+
+|`TechPreviewNoUpgrade`
+a|Turns on Technology Preview features that are not part of the normal supported platform. Turning on this feature set on cannot be undone and prevents upgrades. 
+
+The following Technology Preview features are enabled by this feature set:
+
+* link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation[RotateKubeletServerCertificate]. Enable the rotation of the server TLS certificate on the kubelet.
+* SupportPodPidsLimit. Enables limiting PIDs in Pods.
+
```

Comment:
from https://docs.openshift.com/container-platform/4.5/nodes/clusters/nodes-cluster-enabling-features.html#feature-gate-features_nodes-cluster-enabling ，do we need add Technology Preview features like MachineHealthCheck and LocalStorageCapacityIsolation?

Reply:
@lyman9966 @sttts The list came from here: https://github.com/openshift/api/blob/release-4.6/config/v1/types_feature.go#L29
Stefan, these are the 4 feature sets that the user can enable?

Reply:
@mburke5678 I think you misunderstood me. "https://docs.openshift.com/container-platform/4.5/nodes/clusters/nodes-cluster-enabling-features.html#feature-gate-features_nodes-cluster-enabling" show us there are 4 features affected by feature set "TechPreviewNoUpgrade", such as "RotateKubeletServerCertificate","SupportPodPidsLimit","MachineHealthCheck","LocalStorageCapacityIsolation". Yet your doc lack the latter two.

Reply:
@lyman9966 MachineHealthCheck went GA in 4.3: https://docs.openshift.com/container-platform/4.3/machine_management/deploying-machine-health-checks.html
It appears that `LocalStorageCapacityIsolation` was enabled at some point https://github.com/openshift/api/pull/462/files

I assume this means that the TechPreviewNoUpgrade feature gate would not enable these features.

Reply:
@mburke5678 , Okay, then "MachineHealthCheck" and "LocalStorageCapacityIsolation" doesn't belong to TechPreview feature, so they can be removed.